gdb2cm.go

```diff
@@ -53,33 +56,58 @@ func readDashboardJson(file string) *models.Dashboard {
 	return dbo
 }
 
-func main() {
-	kingpin.Parse()
-	if !strings.HasSuffix(*dashboardFile, ".json") {
-		panic(fmt.Sprintf("%s is not a file...exiting", *dashboardFile))
+func processDir(dbDirGlob, mDir string) {
+	dirs, err := filepath.Glob(dbDirGlob)
+	if err != nil {
+		panic(err)
+	}
+	for _, dir := range dirs {
+		pre := ""
+		if *parentDirAsTeam {
```

Comment:
I get the problem you are solving, but it is kind of gross.  I'm wondering if instead of trying to process entire directories, maybe it would be better if you had a config instead that looks  something like:
```
sources:
  - ['src', 'dest', annotations: {}, labels: {}]
  ...
```

Something like this feels more generic, and flexible with the annoying step of needing to generate this file before running this step.